README.md

```diff
@@ -128,6 +128,12 @@ You can directly extend and run the tests by modifying
 
     tests/test.hs
 
+### Parallelism in Tests
+
+Most tests run in parallel, with a few module dependencies built sequentially in
+advance. Benchmarks are run sequentially after all other tests have finished.
+For details on adding tests, see note [Parallel_Tests] in `tests/test.hs`.
```

Comment:
Note [Parallel_Tests] doesn't exist, but there's a similarly named note. Worth renaming to make it easier to grep.

Also, the windows failure confuses me. I don't know what command the error message refers to.